src/parser/range.rs

```diff
@@ -85,7 +85,6 @@ parser! {
     /// assert!(parser.parse("!").is_err());
     /// # }
     /// ```
-    #[inline]
```

Comment:
Looks unrelated?

Reply:
It was a new warning about duplicate inline so I just removed it as I went.